docs/tutorials/five-minutes-to-feature-flags.md

```diff
@@ -364,6 +364,10 @@ app.listen(3333, () => {
 
 Next restart the node server, and make a new request:
```

Comment:
Could you split this into two separate sentences so that the example commands match the content?

Reply:
@masatakashida could you please make sure these changes are included?